<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatSensorSetForm</class>
 <widget class="QWidget" name="PlatSensorSetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>232</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>硬件配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>通信设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>端口号：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="cbPortList">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="cbComMode">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>串口</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>USB</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>蓝牙</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Wi-Fi</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="pbReloadPort">
             <property name="styleSheet">
              <string notr="true">border: none;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="realtimecurveimage.qrc">
               <normaloff>:/CurveImage/reload_72px_1175851_easyicon.net.png</normaloff>:/CurveImage/reload_72px_1175851_easyicon.net.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>波特率：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="cbBaudRate">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>通信方式：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>传感器参数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="leSensorThick">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>um</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>厚度：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="leSensorWidth">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>宽度：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>长度：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="leSensorLength">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>类型：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cbSensorType">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>1K101</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>实验平台</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>脉冲法</string>
           </property>
           <property name="icon">
            <iconset resource="hardsetimage.qrc">
             <normaloff>:/CurveImage/Pulse.png</normaloff>:/CurveImage/Pulse.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButton_4">
           <property name="text">
            <string>永磁体法</string>
           </property>
           <property name="icon">
            <iconset resource="hardsetimage.qrc">
             <normaloff>:/CurveImage/magnet.png</normaloff>:/CurveImage/magnet.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>扫频法</string>
           </property>
           <property name="icon">
            <iconset resource="hardsetimage.qrc">
             <normaloff>:/CurveImage/frequency.png</normaloff>:/CurveImage/frequency.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>阻抗法</string>
           </property>
           <property name="icon">
            <iconset resource="hardsetimage.qrc">
             <normaloff>:/CurveImage/resistor.png</normaloff>:/CurveImage/resistor.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="hardsetimage.qrc"/>
  <include location="realtimecurveimage.qrc"/>
 </resources>
 <connections/>
</ui>
